app/package.json

```diff
@@ -43,6 +43,7 @@
     "lodash": "^4.17.21",
     "maplibre-gl": "^4.4.1",
     "next": "^14.2.7",
+    "papaparse": "^5.5.1",
```

Comment:
H: The build is failing because the types for papaparse come from a separate package -- running `npm i -d @types/papaparse` or adding `@types/papaparse` to dev dependencies should resolve this

Reply:
👍